### PR TITLE
Add correctness check to fusion tests and make experts identical.

### PR DESCRIPTION
**Pull request by hanhanW**

### CODE REVIEW

**Comment by hanhanW** [comment]
@gysit @MaheshRavishankar could you help review the PR? I don't have permissions to add you to reviewers.

File: python/examples/fusion/test.py [hunk 1]
```diff
@@ -39,47 +43,42 @@ def fill_matmul_fusion():
...
+              'linalg.generic',
+              pad=True,
+          )).then(
+              Tile(
+                  fun_name,
+                  'linalg.matmul',
+                  tile_sizes=[0, 0, 24],
+                  pad=True,
+                  pack_paddings=[1, 1, 0],
+              )).then(Vectorize(fun_name, '', vectorize_paddings=True)).then(
```
Comment (gysit) [inline python/examples/fusion/test.py:70]: I like the approach! It makes sure padding happens in the order we want!

File: python/examples/fusion/definitions.py [hunk 2]
```diff
@@ -165,7 +165,60 @@ def shapes_builder(self, sizes: Mapping[str, Any]) -> List[List[int]]:
...
+    return tensors
+
+  def check_np(self, A: np.dtype, B: np.dtype, C: np.dtype, D: np.dtype,
+               E: np.dtype) -> None:
+    """NumPy checking function.
+
+    Given a list of NumPy values, check the precomputed results matches those of
+    the expected reference implementation.
+    """
+    res = np.dot(A, B) + C
```
Comment (gysit) [inline python/examples/fusion/definitions.py:217]: This is assuming the bias is set to zero right? Using a non zero bias may make sense at some point.

**Review by gysit — APPROVED** [review]
Thanks!

File: python/examples/fusion/definitions.py (hunk 2 above)
Comment (hanhanW) [reply python/examples/fusion/definitions.py:217]: No, I don't think so. I think the bias values are in `C`?

Comment (gysit) [reply python/examples/fusion/definitions.py:217]: OK, I thought the bias is in the 1D array which supposedly is the tensor argument D of check_np?

Comment (hanhanW) [reply python/examples/fusion/definitions.py:217]: It is in `C` which is a 1 D array.

```
    M, N, K = sizes["M"], sizes["N"], sizes["K"]
    return [
        [M, K],
        [K, N],
        [N],
        [M, N],
        [M, N],
    ]
```

`D` is sort of useless. The result of matmul is stored to `D` IIRC.

Comment (gysit) [reply python/examples/fusion/definitions.py:217]: Ok makes sense! Thanks for explaining

**Pull request merged**